portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,94 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/*
+ In order to access the secret page you have to solve a randomly generated equation.
+ Upon the input of a correct answer a button will appear, linking the user to the page.
+ */
+function secretPage(){
+  //Generates a number between 1 and 200 for both numbers.
+  this.a = Math.floor((Math.random()*200)+100);
```

Comment:
this.a will create an instance variable attached to the page. It looks like this variable is only used in this function, so you can declare it as a local variable:

var a = ...
var b = ...

Reply:
Got it